pom.xml

```diff
@@ -252,7 +250,10 @@
                 <artifactId>maven-failsafe-plugin</artifactId>
                 <version>2.20</version>
                 <configuration>
-                    <argLine>-da -Djava.security.manager -Djava.security.policy=${basedir}/src/test/resources/java.policy</argLine>
+                    <argLine>-da -Djava.security.manager
+                        -Djava.security.policy=${basedir}/src/test/resources/java.policy
+                        -Djdk.tls.rejectClientInitiatedRenegotiation=true
```

Comment:
?

Reply:
To remove the warning in the integration logs